requirements.txt

```diff
@@ -0,0 +1 @@
+kubernetes==24.2.0
```

Comment:
For a single requirement (and one as general as kube in this context) could _potentially_ omit requirements.txt and just install it directly in the dockerfile.